ceph.spec.in

```diff
@@ -292,17 +282,27 @@ BuildRequires:	lz4-devel >= 1.7
 %endif
 # distro-conditional make check dependencies
 %if 0%{with make_check}
+%if 0%{?fedora} || 0%{?rhel}
 %if 0%{?fedora} || 0%{?rhel} == 7
 BuildRequires:	libtool-ltdl-devel
+BuildRequires:	python%{_python_buildid}-coverage
```

Comment:
the only, and last, nit: could you move this "BuildRequires" after the next line? so they are ordered alphabetically.